scripts/go_install.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+# Copyright 2018 The Kubernetes Authors.
```

Comment:
2020, or was this copied from another repo?

Reply:
copypasta from https://github.com/kubernetes-sigs/cluster-api-provider-azure/blob/master/scripts/kind-with-registry.sh, which may have been copypasta from elsewhere.

I thought it more appropriate to have the actual year of inclusion vs. inaccurate. 

I'm happy to change it. What should it be?

Reply:
That script has 2020 though?

If the content of this file is net new, we should do 2020.

Reply:
Oops, you’re right. I’ll fix that. Thx!

Reply:
Fixed